libs/confman/inc/public/tfc/confman.hpp

```diff
@@ -77,7 +77,7 @@ class config {
   config(std::shared_ptr<sdbusplus::asio::connection> conn, std::string_view key, storage_type&& def)
       : client_{ conn, key, std::bind_front(&config::string, this), std::bind_front(&config::schema, this),
                  std::bind_front(&config::from_string, this) },
-        storage_{ client_.io_context(), tfc::base::make_config_file_name(key, "json"), std::forward<storage_type>(def) },
+        storage_{ client_.get_io_context(), tfc::base::make_config_file_name(key, "json"), std::forward<storage_type>(def) },
```

Comment:
@jbbjarnason Can you comment on if and why this is needed. Everything still seems to build so maybe this constructor has not been used project wide.

Reply:
this constructor is most likely missing a test of usage